### PR TITLE
Allowed making all services public.

### PR DESCRIPTION
**Pull request by Firtzberg**
[PREF-274](https://55places.atlassian.net/browse/PREF-274)
I've renamed the method and removed the parameter. Let me know if you don't like it.

### CODE REVIEW

File: src/TinyContainerBuilder.php [hunk 1]
```diff
@@ -129,6 +139,12 @@ public function makePublic(string $service): ContainerBuilderInterface
         return $this;
     }
 
+    public function makeAllPublic(): ContainerBuilderInterface
+    {
+        $this->makeAllServicesPublic = true;
```
Comment (arielallon) [inline src/TinyContainerBuilder.php:144]: should we make it a little more "explodey" by first checking that this isn't set and throwing a LogicException if it is?

**Review by arielallon — APPROVED** [review]

**Review by harshpatel991 — APPROVED** [review]

File: src/TinyContainerBuilder.php (hunk 1 above)
Comment (Firtzberg) [reply src/TinyContainerBuilder.php:150]: Updated.

File: src/TinyContainerBuilder.php [hunk 2]
```diff
@@ -124,11 +134,23 @@ public function getInternalContainer(): \Symfony\Component\DependencyInjection\C
 
     public function makePublic(string $service): ContainerBuilderInterface
     {
+        if ($this->makeAllServicesPublic) {
+            throw new \LogicException('Container builder has already been instructed to make all services public.');
+        }
```
Comment (arielallon) [inline src/TinyContainerBuilder.php:139]: @harshpatel991 I want to make sure this won't interfere with the usecase you have in mind given how the code already operates in the wild.

**Review by arielallon — APPROVED** [review]

File: src/TinyContainerBuilder.php (hunk 2 above)
Comment (harshpatel991) [reply src/TinyContainerBuilder.php:139]: Yeah this should be fine

**Review by harshpatel991 — APPROVED** [review]

**Pull request merged**